src/OpenTelemetry.AutoInstrumentation/Configurations/DelayedInitialization.cs

```diff
@@ -25,15 +25,21 @@ internal static class DelayedInitialization
 {
     internal static class Traces
     {
+#if NET462
         [MethodImpl(MethodImplOptions.NoInlining)]
         public static void AddAspNet(LazyInstrumentationLoader lazyInstrumentationLoader, PluginManager pluginManager)
         {
-#if NET462
             new AspNetInitializer(lazyInstrumentationLoader, pluginManager);
-#elif NET6_0_OR_GREATER
-            lazyInstrumentationLoader.Add(new AspNetCoreInitializer(pluginManager));
+        }
 #endif
+
+#if NET6_0_OR_GREATER
```

Comment:
```suggestion
#if !NETFRAMEWORK
```

Reply:
!NETFRAMWORK- whole project potentially should be reviewed. I will keep as it is. (Same comment for all suggestions)